src/main/webapp/less/portlet-grid.less

```diff
@@ -0,0 +1,142 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*---------------------------------------------------
+    PORTLET GRID Mixin   | version: 0.7
+  ---------------------------------------------------
+    A mixin to reflow Bootstrap Classes and to create
```

Comment:
YES!!! it is bootstrap CSS grid recalculation when the portlet is rendered in desktop multi-columns dashboard view. The mobile styles of the table is rendered in large displays when the portlet is rendered in multicolumn. but when the portlet is rendered in a single column layout the regular desktop table styles is rendered.
not sure if I'm clear with this frenglish explanations... :blush:
I'have explained this in my Open Apereo 2016 NY - slides 33/66 ([on the road of multi-tenancy portal](http://fr.slideshare.net/prairycat/on-the-road-of-multi-tenancy-portal-apereo-2016)...)